src/library.h

```diff
@@ -47,6 +47,10 @@ enum library_cb_action
   LIBRARY_CB_DELETE,
 };
 
+#define LIBRARY_SOURCE_FILESCANNER "file-scanner"
```

Comment:
The hyphens look a bit odd. As mentioned, I'm thinking an id might be better, but if not then maybe just set to "file", "rss" and "spotify", since there is a description anyway. Otherwise underscores would be preferable over hyphens, I think.

Reply:
I changed it to `files`, `spotify` and `rss`.